lib/shared/store.js

```diff
@@ -414,24 +426,37 @@ const storeReducer = (state = DEFAULT_STATE, action) => {
         })
       }
 
-      const selectedDrive = findDrive(state, state.getIn([ 'selection', 'drive' ]))
+      const selectedDevices = state.getIn([ 'selection', 'drives' ])
```

Comment:
NIT: should this variable be named `selectedDrives` for consistency?

Reply:
Since it returns a `String[]` containing the `device` fields instead of drive objects, I think it makes more sense with `selectedDevices`.

Reply:
For clarity, perhaps we need an Etcher-wide definition of what's a `device` or a `drive` or a `drive object` or a `driveDevice` etc., and then (try to) use those terms consistently throughout the codebase? Ping @jhermsmeier

Reply:
With this PR I've tried to keep `device : String` and `drive : Object`, as a starting point

Reply:
But it'd still be good to nail down exactly what `device : String` and `drive : Object` actually mean, with examples of what they might contain in different scenarios, on different OSes, etc :)